src/scss/components/_inputs.scss

```diff
@@ -150,13 +150,15 @@ $input-select-margin-right: 1;
     float: left;
     margin-right: 0.5rem;
     @include at-media('tablet') {
-      width: 2rem;
+      width: 2.625rem;
+      padding-top: 0.25rem;
     }
   }
   .usa-radio__label--text,
   .usa-checkbox__label--text {
     @include at-media('tablet') {
       flex: 1;
+      padding-left: 0.5rem;
```

Comment:
Is this new? I don't see the corresponding style in https://github.com/18F/identity-idp/pull/6120

Reply:
Yeah, it was hiden in an index.erb file as `class: "tablet:padding-left-1"`, should be the second file changed in the idp pr.

Reply:
> Not new, it was hiden in an index.erb file as `class: "tablet:padding-left-1"`, should be the second file changed in the idp pr.

Ah! I missed that. Makes sense 👍